contracts/carrot-app/src/handlers/migrate.rs

```diff
@@ -27,9 +28,16 @@ pub struct OldPoolConfig {
     pub asset1: AssetEntry,
 }
 
+#[cw_serde]
+pub struct OldPosition {
```

Comment:
We should version this instead of calling it old. 

I.e. `V0_1Position`

Reply:
Can we use the actual app version here? so think that's 0.3 now?

Reply:
> Can we use the actual app version here? so think that's 0.3 now? @Buckram123

0.1 to 0.3 have the same position structure, so I went with version it got introduced in

Reply:
Also keeping it this way for 0.1->1.0 migration

Reply:
Ok good :)